mava/components/updating/terminators.py

```diff
@@ -97,6 +97,12 @@ def on_parameter_server_run_loop_termination(
         Returns:
             None.
         """
+        if (
+            hasattr(parameter_server, "calculate_absolute_metric")
```

Comment:
Did you try using parameter_server.has("calculate_absolute_metric")? So that we keep the consistency with using that custom "has" function Sasha implemented.

Reply:
The issue that `has` works only in the case of instance of a class however `calculate_absolute_metric` is an attribute so we can't use has in this case.